src/storhaug

```diff
@@ -324,15 +324,28 @@ setup_cluster()
 
     pcs cluster auth ${HA_SERVERS} -u hacluster -p ${HA_PASSWORD} --force
     pcs cluster setup --force --name ${HA_NAME} ${HA_SERVERS} || storlog "ERR" "Failed to setup cluster ${HA_NAME}"
+
+    # BZ 1284404, 1425110, allow time for SSL certs to propagate, until then
```

Comment:
Don't reference BZ numbers unless you're specifying what bugzilla system they belong to.